python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -766,10 +772,11 @@ def _process_commands(self, commands, project, entities):
 
         for command in commands:
             if command["app_name"] is not None:
+                self.logger.debug("Keeping command %s -- it has an associated app." % command)
```

Comment:
Did you know you could use a comma instead of `%`when formatting the string?

`self.logger.debug("This %s my %s.", "is", "boomstick")`


This has two benefits:

The first, which isn't that important, is that formatting will be done only if the message needs to be written out. For debug messages, this means we can skip formatting and shave of some execution time. Not a big deal here, I know.

The second, which is more important, is that if you are missing some arguments in your `% (...)` the code will raise an exception. If you use comma separate the arguments, the Python logger will instead catch the exception itself and log an error saying which line in your code was missing some arguments. Considering we sometimes log stuff for things we don't have the greatest coverage for, its safer to log using commas knowing that if we never end up in that edge case and need to log something we can be certain a formatting error won't disrupt the app just because of a logging mistake.